sources/packages/api/src/services/user/user.model.ts

```diff
@@ -0,0 +1,5 @@
+export class UserLoginInfo {
+  id: number;
+  isActive: boolean;
+  studentId?: number;
+}
```

Comment:
Will it be better to make this an interface since it is not a DTO that needs to be rendered in swagger and also it looks like this model was created for type-checking?

Reply:
That was by mistake. I will change it.

Reply:
Fixed!